src/screens/CodeVerifyScreen/style.ts

```diff
@@ -6,7 +6,7 @@ import {fontSizes} from "../../../assets/fontSizes/fontSizes";
 export const styles = StyleSheet.create({
     container: {
         flex: 1,
-        backgroundColor: colors.WHITEBlUE
+        backgroundColor: colors.WHITEBLUE
     },
     content: {
         flex: 1,
```

Comment:
No need to create separate files for styles. In react-native there's no such thing as styles cascading, so each components has exactly the styles you pass to it. It's more convenient to have styles in the same file they're used